sites/docs/src/pages/_ViewConfigEditor.js

```diff
@@ -114,15 +119,52 @@ export default function ViewConfigEditor(props) {
     setUrl(nextUrl);
   }
 
+  function sanitiseURLs(urls) {
+    return urls
+      .replace(/ /g, '')
+      .split(/;/)
+      .filter(url => url.match(/^http/g));
```

Comment:
```suggestion
    return urls
      .split(/;/)
      .map(url => url.trim())
      .filter(url => url.match(/^http/g));
```

I think we only want to remove whitespace at the start/end to allow URLs that contain spaces